tools/scripts-config-eslint/.eslintrc.json

```diff
@@ -8,6 +8,7 @@
     "plugin:import/recommended",
     "plugin:jsx-a11y/recommended",
     "plugin:react/recommended",
+    "plugin:react/jsx-runtime",
```

Comment:
https://github.com/jsx-eslint/eslint-plugin-react
THere is a lots of rule, why do you want to remove all of them ?
I am sure there is update here that take this into account.

Reply:
indeed